### PR TITLE
Fix #181577

### LINKED ISSUE #181577: na.finalfantasyxiv.com
### Issue URL (Annoyance)
 [https://na.finalfantasyxiv.com/lodestone/character/2467236/blog/403635/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fna.finalfantasyxiv.com%2Flodestone%2Fcharacter%2F2467236%2Fblog%2F403635%2F)

### Screenshots
<details>
  <summary>Screenshot 1 </summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2 </summary>

  Screenshot 2
</details>
<details>
  <summary>Screenshot 3 </summary>

  Screenshot 3
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard for Safari v1.11.18
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Russian, <br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by forcomprehension**
Fix #181577

### CODE REVIEW

File: SocialFilter/sections/general_elemhide.txt [hunk 1]
```diff
@@ -972,6 +972,7 @@ designboom.com,livescience.com,pbtech.co.nz,peoplesworld.org,space.com##.fa-face
 ##.social__likes
 ##.social_bar
 ##.social_btn
+##.social__btn
```
Comment (Sergey-Lyapin) [inline SocialFilter/sections/general_elemhide.txt:975]: Please make a specific rule, there is no sense for making a general rule.
`finalfantasyxiv.com##.l__blogsocial`

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt [hunk 2]
```diff
@@ -1578,7 +1578,7 @@ produkte.erstegroup.com##.OverlayDisclaimer
 ||paih.gov.pl/_js/2021/jquery.magnific-popup.js
 evropochta.by#%#//scriptlet('set-local-storage-item', 'saveCookie', 'true')
 evropochta.by#%#//scriptlet('set-local-storage-item', 'otherCookie', 'false')
-telesign.com##.osano-cm-window__widget
+finalfantasyxiv.com,telesign.com##.osano-cm-window__widget
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/Cookies/sections/cookies_specific.txt:1581]: I can't reproduce the cookie popup. It is blocked by `||cmp.osano.com^`

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt (hunk 2 above)
Comment (forcomprehension) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt:1581]: i can reproduce this chrome

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt:1581]: Do you have AdGuard Annoyances filter enabled?

@Alex-302 @jellizaveta please take a look

Comment (forcomprehension) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt:1581]: you're right. when i turned on annoyances, this widget disappeared. Should i discard this change?

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt:1581]: yes, please discard

**Review by Sergey-Lyapin — APPROVED** [review]

**Comment by Sergey-Lyapin** [comment]
@jellizaveta please review

**Review by jellizaveta — APPROVED** [review]

**Pull request merged**